### PR TITLE
idm_22.2.1_release

### PR DESCRIPTION
**Pull request by manjunathdhegde-2910**
Signed-off-by: Manjunath Hegde <Manjunath.Hegde@oracle.com>

### CODE REVIEW

**Review by pratdash-orcl — COMMENTED** [review]
Looks fine to me.

**Review by pratdash-orcl — DISMISSED** [review]
Looks fine to me. Approving it.

File: OracleInternetDirectory/imagetool/12.2.1.4.0/README.md [hunk 1]
```diff
@@ -1,208 +1,17 @@
...
-5. [Required build files](#5-required-build-files)
-6. [Additional build commands](#6-additional-build-commands)
-7. [Steps to Create Oracle Internet Directory image](#7-steps-to-create-oracle-internet-directory-image)
-8. [Generate Sample dockerfile with imagetool](#8-generate-sample-dockerfile-with-imagetool)
+For instructions on building an OID image with WebLogic Image Tool see [Create or update image](https://oracle.github.io/fmw-kubernetes/oid/create-or-update-image#create-or-update-an-oracle-internet-directory-services-manager-image-using-the-weblogic-image-tool).
 
-## 1. Introduction
 
-This README describes the steps involved in building an Oracle Internet Directory image with the WebLogic Image Tool.
+# Licensing & Copyright
```
Comment (Djelibeybi) [inline OracleInternetDirectory/imagetool/12.2.1.4.0/README.md:8]: Please remove this heading as it is redundant.

File: OracleInternetDirectory/imagetool/12.2.1.4.0/README.md [hunk 2]
```diff
@@ -1,208 +1,17 @@
 # Building an Oracle Internet Directory image with WebLogic Image Tool
 
-## Contents
+# Introduction
```
Comment (Djelibeybi) [inline OracleInternetDirectory/imagetool/12.2.1.4.0/README.md:3]:
```suggestion
## Documentation
```

File: OracleInternetDirectory/imagetool/12.2.1.4.0/README.md [hunk 3]
```diff
@@ -1,208 +1,17 @@
...
 
-1. [Introduction](#1-introduction)
-2. [Prerequisites](#2-prerequisites)
-3. [Setup WebLogic Image Tool](#3-setup-weblogic-image-tool)
-4. [Download the required packages and installers](#4-download-the-required-packagesinstallers-and-patches)
-5. [Required build files](#5-required-build-files)
-6. [Additional build commands](#6-additional-build-commands)
-7. [Steps to Create Oracle Internet Directory image](#7-steps-to-create-oracle-internet-directory-image)
-8. [Generate Sample dockerfile with imagetool](#8-generate-sample-dockerfile-with-imagetool)
+For instructions on building an OID image with WebLogic Image Tool see [Create or update image](https://oracle.github.io/fmw-kubernetes/oid/create-or-update-image#create-or-update-an-oracle-internet-directory-services-manager-image-using-the-weblogic-image-tool).
```
Comment (Djelibeybi) [inline OracleInternetDirectory/imagetool/12.2.1.4.0/README.md:5]:
```suggestion
Detailed instructions on how to [create or update an Oracle Internet Directory container image](https://oracle.github.io/fmw-kubernetes/oid/create-or-update-image#create-or-update-an-oracle-internet-directory-services-manager-image-using-the-weblogic-image-tool) using the WebLogic Image Tool can be found in the [Oracle Fusion Middleware on Kubernetes](https://oracle.github.io/fmw-kubernetes/) documentation.
```

However, the URL to create-or-update-image does not exist and there does not appear to be any open PR that would create it.

File: OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md [hunk 4]
```diff
@@ -1,272 +1,10 @@
 Building an Oracle Unified Directory image with WebLogic Image Tool
 ===================================================================
 
-## Contents
+# Introduction
```
Comment (Djelibeybi) [inline OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md:4]:
```suggestion
## Documentation
```

File: OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md [hunk 5]
```diff
@@ -1,272 +1,10 @@
 Building an Oracle Unified Directory image with WebLogic Image Tool
 ===================================================================
```
Comment (Djelibeybi) [inline OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md:2]:
```suggestion
# Building an Oracle Unified Directory image with WebLogic Image Tool
```

File: OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md [hunk 6]
```diff
@@ -1,272 +1,10 @@
...
 
-1. [Introduction](#1-introduction)
-2. [Prerequisites](#2-prerequisites)
-3. [Setup WebLogic Image Tool](#3-setup-weblogic-image-tool)
-4. [Download the required packages/installers](#4-download-the-required-packagesinstallers-patches)
-5. [Required build files](#5-required-build-files)
-6. [Additional build commands](#6-additional-build-commands)
-7. [Steps to Create Oracle Unified Directory image](#7-steps-to-create-oracle-unified-directory-image)
-8. [Generate Sample dockerfile with imagetool](#8-generate-sample-dockerfile-with-imagetool)
+For instructions on building an OUD image with WebLogic Image Tool see [Create or update image](https://oracle.github.io/fmw-kubernetes/oud/create-or-update-image#create-or-update-an-oracle-unified-directory-image-using-the-weblogic-image-tool).
```
Comment (Djelibeybi) [inline OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md:6]:
```suggestion
Detailed instructions on how to [create or update an Oracle Unified Directory container image](https://oracle.github.io/fmw-kubernetes/oud/create-or-update-image#create-or-update-an-oracle-unified-directory-image-using-the-weblogic-image-tool) using the WebLogic Image Tool can be found in the [Oracle Fusion Middleware on Kubernetes](https://oracle.github.io/fmw-kubernetes/) documentation.
```

However, the URL to create-or-update-image does not exist and there does not appear to be any open PR that would create it.

File: OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md [hunk 7]
```diff
@@ -1,272 +1,10 @@
...
-
-# 8. Generate Sample dockerfile with imagetool
-
-If you want to review a sample dockerfile created with the imagetool issue the `imagetool` command with the `--dryRun` option:
-
-```
-./imagetool.sh @<work directory>/docker-images/OracleUnifiedDirectory/imagetool/12.2.1.4.0/additionalBuildCmds.txt --dryRun
-```
 
 # Licensing & Copyright
```
Comment (Djelibeybi) [inline OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md:9]:
```suggestion
## License
```

File: OracleUnifiedDirectorySM/dockerfiles/12.2.1.4.0/container-scripts/get_healthcheck_url.sh [hunk 8]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#
+#Copyright (c) 2020, 2022, Oracle and/or its affiliates.
```
Comment (Djelibeybi) [inline OracleUnifiedDirectorySM/dockerfiles/12.2.1.4.0/container-scripts/get_healthcheck_url.sh:3]:
```suggestion
# Copyright (c) 2022, Oracle and/or its affiliates.
```

File: OracleUnifiedDirectorySM/imagetool/12.2.1.4.0/README.md [hunk 9]
```diff
@@ -1,289 +1,18 @@
 Building an Oracle Unified Directory Services Manager Image with Weblogic Image Tool
 ====================================================================================
```
Comment (Djelibeybi) [inline OracleUnifiedDirectorySM/imagetool/12.2.1.4.0/README.md:2]:
```suggestion
# Building an Oracle Unified Directory Services Manager Image with Weblogic Image Tool
```

File: OracleUnifiedDirectorySM/imagetool/12.2.1.4.0/README.md [hunk 10]
```diff
@@ -1,289 +1,18 @@
...
 
-## Contents
+# Introduction
```
Comment (Djelibeybi) [inline OracleUnifiedDirectorySM/imagetool/12.2.1.4.0/README.md:4]:
```suggestion
## Documentation
```

File: OracleUnifiedDirectorySM/imagetool/12.2.1.4.0/README.md [hunk 11]
```diff
@@ -1,289 +1,18 @@
...
 
-1. [Introduction](#1-introduction)
-2. [Prerequisites](#2-prerequisites)
-3. [Setup WebLogic Image Tool](#3-setup-weblogic-image-tool)
-4. [Download the required packages/installers](#4-download-the-required-packagesinstallers)
-5. [Additional build files](#5-additional-build-files)
-6. [Additional build commands](#6-additional-build-commands)
-7. [Steps to Create Oracle Unified Directory Services Manager image](#7-steps-to-create-oracle-unified-directory-services-manager-image)
-8. [Generate Sample dockerfile with imagetool](#8-generate-sample-dockerfile-with-imagetool)
+For instructions on building an OUDSM image with WebLogic Image Tool see [Create or update image](https://oracle.github.io/fmw-kubernetes/oudsm/create-or-update-image#create-or-update-an-oracle-unified-directory-services-manager-image-using-the-weblogic-image-tool).
```
Comment (Djelibeybi) [inline OracleUnifiedDirectorySM/imagetool/12.2.1.4.0/README.md:6]:
```suggestion
Detailed instructions on how to [create or update an Oracle Unified Directory Services Manager container image](https://oracle.github.io/fmw-kubernetes/oudsm/create-or-update-image#create-or-update-an-oracle-unified-directory-services-manager-image-using-the-weblogic-image-tool) using the WebLogic Image Tool can be found in the [Oracle Fusion Middleware on Kubernetes](https://oracle.github.io/fmw-kubernetes/) documentation.
```

However, the provided URL returns a 404 error.

**Review by Djelibeybi — CHANGES_REQUESTED** [review]

File: OracleInternetDirectory/imagetool/12.2.1.4.0/README.md (hunk 3 above)
Comment (manjunathdhegde-2910) [reply OracleInternetDirectory/imagetool/12.2.1.4.0/README.md:5]: The URL link will be available once we have it published. It is still undergoing last minute checks.

File: OracleUnifiedDirectorySM/imagetool/12.2.1.4.0/README.md (hunk 11 above)
Comment (manjunathdhegde-2910) [reply OracleUnifiedDirectorySM/imagetool/12.2.1.4.0/README.md:6]: The URL link will be available once we have it published. It is still undergoing last minute checks.

File: OracleInternetDirectory/imagetool/12.2.1.4.0/README.md (hunk 2 above)
Comment (manjunathdhegde-2910) [reply OracleInternetDirectory/imagetool/12.2.1.4.0/README.md:3]: changed

File: OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md (hunk 4 above)
Comment (manjunathdhegde-2910) [reply OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md:4]: changed

File: OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md (hunk 5 above)
Comment (manjunathdhegde-2910) [reply OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md:2]: changed

File: OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md (hunk 7 above)
Comment (manjunathdhegde-2910) [reply OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md:9]: changed

File: OracleUnifiedDirectorySM/dockerfiles/12.2.1.4.0/container-scripts/get_healthcheck_url.sh (hunk 8 above)
Comment (manjunathdhegde-2910) [reply OracleUnifiedDirectorySM/dockerfiles/12.2.1.4.0/container-scripts/get_healthcheck_url.sh:3]: changed

File: OracleUnifiedDirectorySM/imagetool/12.2.1.4.0/README.md (hunk 10 above)
Comment (manjunathdhegde-2910) [reply OracleUnifiedDirectorySM/imagetool/12.2.1.4.0/README.md:4]: changed

**Comment by Djelibeybi** [comment]
I have switch this PR to draft mode as the documentation is not ready yet. When you have opened the PR for the documentation, you can mark this as ready to review. Please provide a link to the PR when you do.

**Comment by manjunathdhegde-2910** [comment]
The following documentation PRs are merged.
https://github.com/oracle/fmw-kubernetes/pull/107
https://github.com/oracle/fmw-kubernetes/pull/106
The documentation links are accessible now. Its ready for review now.

File: OracleUnifiedDirectorySM/imagetool/12.2.1.4.0/README.md [hunk 12]
```diff
@@ -1,289 +1,15 @@
-Building an Oracle Unified Directory Services Manager Image with Weblogic Image Tool
+# Building an Oracle Unified Directory Services Manager Image with Weblogic Image Tool
 ====================================================================================
```
Comment (Djelibeybi) [inline OracleUnifiedDirectorySM/imagetool/12.2.1.4.0/README.md:2]:
```suggestion
# Building an Oracle Unified Directory Services Manager Image with Weblogic Image Tool
```

File: OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md [hunk 13]
```diff
@@ -1,280 +1,16 @@
-Building an Oracle Unified Directory image with WebLogic Image Tool
+# Building an Oracle Unified Directory image with WebLogic Image Tool
 ===================================================================
```
Comment (Djelibeybi) [inline OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md:2]:
```suggestion
# Building an Oracle Unified Directory image with WebLogic Image Tool
```

**Review by Djelibeybi — APPROVED** [review]

**Pull request merged**